src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface Area {
```

Comment:
Don't begin interface implementation with an empty line. Remove the redundant empty line at the beginning of the interface definition.